azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -212,6 +220,18 @@ public KubernetesContainerizedImpl(final Props azkProps,
         this.azkProps
             .getString(ContainerizedDispatchManagerProperties.KUBERNETES_MOUNT_PATH_FOR_JOBTYPES,
                 DEFAULT_APP_MOUNT_PATH_PREFIX_FOR_JOBTYPES);
+    this.dependencyTypes =
```

Comment:
Can you please mention format to provide these properties? Specially in case of mount path for multiple dependencies. What is the delimiter and all?